libsql/src/sync.rs

```diff
@@ -185,9 +190,11 @@ impl SyncContext {
             self.sync_url,
             generation,
             frame_no,
-            frame_no + 1
+            // the server expects the range of [start, end) frames, i.e. end is exclusive
+            // so we add +1 so that we can pull in the batch size of `self.pull_batch_size`
+            frame_no + self.pull_batch_size
```

Comment:
According to the comment, it says "add +1", but I don't see that.

Reply:
@khuezy thanks for taking a look! I am still experimenting with the batch sizes, so the comment is not up to date.